Android/app/src/main/res/values/strings.xml

```diff
@@ -126,6 +127,7 @@
     <string name="text_joining_date">Joined on %1$s</string>
     <string name="profile_picture_updated">Profile Picture Updated!</string>
     <string name="text_source">"Source : "</string>
+    <string name="no_friends_message">You have no friends</string>
```

Comment:
This might sound sad to a person.
Let's change the message to `You haven't added any trip with friends yet`

Reply:
okay :P